kernelci/cli.py

```diff
@@ -189,6 +189,20 @@ class Args(object):
         'help': "Path the output directory",
     }
 
+    install_path = {
+        'name': '--install-path',
+        'required': False,
```

Comment:
This works, but it's not really necessary to add here as arguments put in `opt_args` in the command class have required set to `False` automatically.